test/string-checker.js

```diff
@@ -297,29 +339,20 @@ describe('modules/string-checker', function() {
 
     describe('error filter', function() {
         beforeEach(function() {
-            checker = new Checker();
+            checker = new StringChecker();
             checker.registerDefaultRules();
         });
 
         it('should accept a path to a filter function to filter out errors', function() {
             checker.configure({
                 disallowQuotedKeysInObjects: true,
-                errorFilter: __dirname + '/data/error-filter.js'
+                errorFilter: require(__dirname + '/data/error-filter.js')
```

Comment:
This is `StringChecker` test and for some reason `StringChecker` was not actually tested here. `Checker` was. I have changed `new Checker(...)` to `new StringChecker(...)` and it required to change some tests for `StringChecker` compatibility.

> Does this require a change in the way users use this feature?

Nope.